### DISCUSSION TITLE
Bonito promethION models

### DISCUSSION THREAD

**Discussion by sengelen** [nanoporetech/bonito#106] [Q&A]
Hi,
I would like to basecall data from promethION with bonito . Which model can I use for promethION r9.4.1 data ? 
Stefan

**Comment by iiSeymour** [comment]
Hey @sengelen 

The `dna_r9.4.1` model is suitable to use on both MinION and PromethION.

**Reply by sengelen** [reply]
Thank you.
I had a doubt because the guppy promethION models have a "_prom" in there names.
I would also like to know if this bonito model is integrated in lastest guppy version (4.4.*) and if so what is its name (dna_r9.4.1_450bps_hac_prom) ?
